Distribution/Server/Framework/RequestContentTypes.hs

```diff
@@ -2,7 +2,6 @@
 -- |
 --
 -----------------------------------------------------------------------------
--- |
```

Comment:
This was the wrong `-- |` to delete.  (The parser for the module header (license etc.) is very fragile.)
I am fixing this.